### DISCUSSION TITLE
v0.72.1

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#5954] [Announcements]
### Features and improvements

* Commenting in code blocks is now possible in https://github.com/outline/outline/pull/5953

### Fixes

* Fixed a **memory leak** in the editor causing performance issues – this affected all documents, but particularly those with many images in https://github.com/outline/outline/pull/5950
* Fixed links to attachments are incorrect in emailed notifications in https://github.com/outline/outline/pull/5935
* Fixed cookie not correctly removed in browser when logging out
* Fixed translations are cached too hard causing temporary incorrect translations on Outline upgrade
* Fixed sidebar does not collapse immediately on control click
* Microsoft logo on login screen now inherits theme colors
* "Untitled" text is now correctly translated

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.72.1'>v0.72.1</a>.</em>

**Comment by YokukYo** [comment]
Sorry, I didn't know where to post this.
--------------------------------------------------------------------
main -> package.json "version": "0.72.0" No change.

**Reply by YokukYo** [reply]
I compiled it using the github action and it came up with 0.72.
